src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Assistant/TicketAssistantTile.tsx

```diff
@@ -0,0 +1,97 @@
+import {StyleSheet, useTheme} from '@atb/theme';
+import {StaticColor, getStaticColor} from '@atb/theme/colors';
+import {View} from 'react-native';
+import {TouchableOpacity} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import React from 'react';
+import {translation as _, useTranslation} from '@atb/translations';
+
+type TicketAssistantProps = {
+  accented?: boolean;
+  onPress: () => void;
+  testID: string;
+};
+const TicketAssistantTile: React.FC<TicketAssistantProps> = ({
+  accented,
+  onPress,
+  testID,
+}) => {
+  const styles = useStyles();
+  const {themeName} = useTheme();
+  const color: StaticColor = accented ? 'background_accent_3' : 'background_0';
+  const themeColor = getStaticColor(themeName, color);
+  const {t} = useTranslation();
+
+  const texts = {
```

Comment:
Could these texts go into TicketAssistantTexts file?

Reply:
Added it to the ticketing.ts texts file, is that ok?